### PR TITLE
Bump to version 1.1

### PR DESCRIPTION
**Pull request by afalko**

### CODE REVIEW

**Review by avimanyum — APPROVED** [review]

**Comment by justinharringa** [comment]
Needs a bit more love: https://github.com/salesforce/dockerfile-image-update/blob/master/Makefile#L5

You can see the proposed next version here: https://github.com/salesforce/dockerfile-image-update/runs/4182148984?check_suite_focus=true

Sorry, I hadn't left this as dynamic as it should be. 😄

File: .ci.deploy.sh [hunk 1]
```diff
@@ -32,6 +32,7 @@ docker run --rm -v "${PWD}":/usr/src/build \
                 -e CI_DEPLOY_PASSWORD \
                 -e GPG_KEY_NAME \
                 -e GPG_PASSPHRASE \
+                -e MVN_SNAPSHOT_VERSION \
```
Comment (justinharringa) [inline .ci.deploy.sh:35]: What is this supposed to do? Nothing is consuming it

File: Makefile [hunk 2]
```diff
@@ -34,7 +34,7 @@ get-itest-jar-from-maven-image:
 
 get-new-version-from-tag:
 	# Get the latest patch revision
-	export LATEST_PATCH_VERSION=$$(git describe --match "dockerfile-image-update-1.0.*" --abbrev=0 --tags | sed s/dockerfile-image-update-1.0.// $<); \
+	export LATEST_PATCH_VERSION=$$(git describe --match "dockerfile-image-update-1.1.*" --abbrev=0 --tags | sed s/dockerfile-image-update-1.1.// $<); \
 	NEW_PATCH_VERSION=1.0.$$(($${LATEST_PATCH_VERSION} + 1)); \
```
Comment (justinharringa) [inline Makefile:38]:
```suggestion
	NEW_PATCH_VERSION=1.1.$$(($${LATEST_PATCH_VERSION} + 1)); \
```

Comment (justinharringa) [reply Makefile:38]: https://github.com/salesforce/dockerfile-image-update/runs/4183628572?check_suite_focus=true#step:3:11

File: .ci.deploy.sh (hunk 1 above)
Comment (justinharringa) [reply .ci.deploy.sh:35]:
```suggestion
```

**Comment by justinharringa** [comment]
`New patch version: 1.1.1`
https://github.com/salesforce/dockerfile-image-update/runs/4183844087?check_suite_focus=true#step:3:11

**Review by justinharringa — APPROVED** [review]

**Pull request merged**